extensions/amp-story/0.1/amp-story-unsupported-browser-layer.css

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-unsupported-browser-overlay {
+  position: fixed !important;
+  z-index: 200001 !important; /** no-rotation-overlay + 1 */
```

Comment:
Does this actually apply inside the shadow DOM?  Or does the z-index need to be specified on the host?

Reply:
I had no idea, but it actually works just like if there was no shadow DOM:
https://codepen.io/gmajoulet/pen/mxxLME